tests/ha/hawk_gui.pm

```diff
@@ -49,6 +49,7 @@ sub run {
     my $docker_image = "registry.opensuse.org/home/rbranco/branches/opensuse/templates/images/tumbleweed/containers/hawk_test";
     assert_script_run("docker pull $docker_image", 240);
 
+    record_soft_failure('bsc#1167633 - No desktop session in tty2 on SLED') if is_sle('15-SP2+') && check_var('SLE_PRODUCT', 'sled');
```

Comment:
Don't like too much soft failing without verifying the condition, but I guess we can add that at a later date.

Reply:
We can also add the record_soft_failure later :)

Please change to not include an unconditional record_soft_failure